backend/tournesol/views/preview.py

```diff
@@ -356,13 +405,16 @@ def get(self, request, uid):
         except ConnectionError:
             return self.default_preview()
 
-        youtube_thumbnail = youtube_thumbnail.resize(
-            numpy.multiply((320, 180), upscale_ratio)
-        )
+        # (width, height, left, top)
```

Comment:
Is it common to store bounding boxes like that? I would have put the position first. But anyway I think using a bbox to store these value makes the code more difficult to read an maintain. You always have to remember where each value is in the tuple, and expressions like `youtube_thumbnail_bbox[2:4]` are not easy to interpret for us humans. I would rather have a position and a size.

And if we keep a bounding box I would rather use Pillow's convention which seems to be `[left, top, right, bottom]` ([[1]](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.textbbox), [[2]](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.rectangle)) since that's the tool we use to draw things.